firefox-ios/Client/Application/AppDelegate.swift

```diff
@@ -233,6 +235,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             self?.backgroundTabLoader.loadBackgroundTabs()
         }
     }
+
+    private func updateWallpaperMetadata() {
+        wallpaperMetadataQueue.async {
+            let wallpaperManager = WallpaperManager()
```

Comment:
@adudenamedruby I kept this code basically the same as it was before, but wanted to double check if we should really be creating a new manager instance for each of these checks?

Reply:
Yeah, the `WallpaperManager` was written very much with the intention that it gets created, does a thing, and then gets discarded. If things don't need to be around, I personally don't like forcing them to hang out like introverts at a party.